### PR TITLE
BZ1704377: Add a note

### PR DESCRIPTION
**Pull request by xenolinux**
OCP version: 3.11
[Preview](https://deploy-preview-42335--osdocs.netlify.app/openshift-enterprise/latest/admin_guide/assembly_replace-etcd-member.html#adding-etcd-host-ansible_replace-etcd-member)
[Bug](https://bugzilla.redhat.com/show_bug.cgi?id=1704377)
QE contact @geliu2016 LGTMed

### CODE REVIEW

File: day_two_guide/topics/proc_scaling-etcd-ansible.adoc [hunk 1]
```diff
@@ -36,6 +36,11 @@ master-2.example.com
 etcd0.example.com <1>
 ----
 <1> Add these lines.
++
+[NOTE]
+====
+Replace the old etcd host entry with the new etcd host entry in the inventory file. While replacing the older etcd host, you must create a copy of `/etc/etcd/ca/` directory. Alternatively, you can redeploy etcd ca and certs before scaling up the etcd hosts.
```
Comment (mjpytlak) [inline day_two_guide/topics/proc_scaling-etcd-ansible.adoc:42]: Should etcd host be `etcd host`?

Comment (xenolinux) [reply day_two_guide/topics/proc_scaling-etcd-ansible.adoc:42]: umm, I checked for similar entries in the docs. For eg etcd member, etcd backup etc. It's written in plain format. So I think we should keep etcd host in the same manner? Please correct me if I'm wrong.

Comment (mjpytlak) [reply day_two_guide/topics/proc_scaling-etcd-ansible.adoc:42]: Right. But in this case etcd host appears to be a value in the Ansible inventory file referenced directly above your note.

Comment (xenolinux) [reply day_two_guide/topics/proc_scaling-etcd-ansible.adoc]: You are correct. Thanks for the clarification. I addressed the change.

**Comment by mjpytlak** [comment]
@xenolinux peer review LGTM.

**Pull request merged**